examples/kitchen-sink/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "MONOREPO_DEV=true tinacms dev -c \"next dev\"",
+    "dev2": "tinacms dev -c \"next dev\"",
```

Comment:
commit?

Reply:
Yeah it's nice to debug without the `MONOREPO_DEV` variable, but usually during dev that's what you want.